Network Design Guide/avs-networking-basics.md

```diff
@@ -1,41 +1,42 @@
+## What is the role played by ExpressRoute?
+Azure VMware Solution runs on bare-metal VMWare vSphere ESXi nodes deployed in Azure datacenters and attached to a physical network. Just like how ExpressRoute circuits allow Azure customers to establish layer-3 connectivity between their physical datacenter networks and Azure VNets, a dedicated ExpressRoute implementation provides layer-3 connectivity between physical ESXi nodes and Azure VNets. 
+When an Azure VMware Solution Private Cloud is provisioned, an associated ExpressRoute circuit is also instantiated in a Microsoft-managed subscription. The private cloud’s owner can then connect the circuit to one or more ExpressRoute Virtual Network Gateways in Azure VNets, by redeeming authorization keys for the circuit (the same procedure used to create connections between ExpressRoute Gateways and customer-managed circuits). Please refer to the [Azure VMware Solution official documentation](https://learn.microsoft.com/azure/azure-vmware/deploy-azure-vmware-solution?tabs=azure-portal#connect-to-azure-virtual-network-with-expressroute ) for detailed instructions.
  
 ![figure2](media/figure2.png) 
-Figure 2. Azure VMware Solution leverages a dedicated Expressroute implementation to provide layer-3 connectivity between Azure VNets and the physical network to which the VMWare ESXi clusters are attached. The VMware ESXi clusters are hosted in the same Microsoft datacenter facilities that host the Azure platform.
+Figure 2. Azure VMware Solution leverages a dedicated ExpressRoute implementation to provide layer-3 connectivity between Azure VNets and the physical network to which the VMWare ESXi clusters are attached. The VMware ESXi clusters are hosted in the same Microsoft datacenter facilities that host the Azure platform.
 
-## What is the role played by Expressroute Global Reach?
-An Azure Expressroute Gateway cannot be used to route traffic between on-prem locations connected to it over different circuits. This limitation applies to the Azure VMware Solution dedicated Expressroute implementation too, as shown in the figure below.
+## What is the role played by ExpressRoute Global Reach?
+An Azure ExpressRoute Gateway cannot be used to route traffic between on-premises locations connected to it over different circuits. This limitation applies to the Azure VMware Solution dedicated ExpressRoute implementation too, as shown in the figure below.
  
 ![figure3](media/figure3.png) 
-Figure 3. ExpressRoute does not support routing traffic between different circuits connected to the same gateway.
+Figure 3. ExpressRoute does not support routing traffic between different circuits connected to the same Gateway.
```

Comment:
Here "gateway" is used as a generic term, just like "circuit" in the same sentence. I think it can stay lower-case. I would use the capitalized version only if the official resource name "Virtual Network Gateway" is used....

Reply:
ah, makes sense. Thanks for the clarification!